src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -371,6 +371,7 @@ Flags:
     --all_account_details    <boolean>       (optional)          Set a flag for returning all account details.
     --all_bucket_details     <boolean>       (optional)          Set a flag for returning all bucket details.
     --all_connection_details <boolean>       (optional)          Set a flag for returning all connection details.
+    --lifecycle              <boolean>       (optional)          Set a flag for returning lifecycle details on the current host.
```

Comment:
You added a flag here, it should be reflected in the file `docs/NooBaaNonContainerized/Health.md` as well.

Reply:
I'm aware, unfortunately due to short timelines documentation is our last priority right now, it's mentioned as a gap in [8861](https://github.com/noobaa/noobaa-core/issues/8861)

Reply:
I would explicitly mention in under "Add CLI documentation" so we will not forget it.

Reply:
please add it if you feel it's needed

Reply:
I added, thanks.